README.md

```diff
@@ -0,0 +1,146 @@
+# Terraform AWS Application Cluster
+
+## Basic Information
+
+This terraform module can be used to create an application-cluster consisting of
++ ApplicationLoadBalancer
+    + 1 Listener for Port 80
+    + 1 Listener for Port 443 (An AWS-certificate is mandatory for this at the moment)
++ AutoScaling-Group
++ IAM-Instance-Profile
++ LaunchConfiguration
++ TargetGroup 
+
+## Usage
+### Parameters
+```hcl-terraform
+variable "application_cluster_elb_security_groups" {
```

Comment:
suggest to use 'lb' instead of 'elb' because 'ApplicationLoadBalancer' != 'ElasticLoadBalancer'! Also used in other variables below